WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,227 @@
+  There are several reference in this WEP to the **core operations (CoreOps) team**. This team is currently known as the **core team** and the team description can be found on the `Core team page <https://www.writethedocs.org/team/#core-team>`_. The updated name is intended to reflect the purpose of the team more accurately in the context of its duties for the community. 
+
+WEP team
+  The **WEP team** is a new team that reviews WEP proposals for clarity and completeness. For the purpose of this WEP the team is initially composed of the Core Operations team, but in the the future will be open for other community members to join. 
+  
+Note that the process to join and leave teams is out of scope of this WEP and will be handled in a future WEP.
+
+At a very high level, the WEP submission process includes the following stages:
+
+#. `Pre-proposal`_ — Someone has an idea and they start collecting early input and feedback to see if it is within the scope of a WEP. This can be done informally, publicly or privately.
+
+#. `Draft submission and pre-review`_ — The WEP author writes a rough draft of the WEP and submits it in a pull request (PR) to the WEPs repository. The draft is pre-reviewed by an editor for style, formatting, and eligibility for the WEP process. If the WEP is pre-approved, it is accepted for discussion and the 30-day review period begins.
+
+#. `Community review, discussion, and updates`_ — During the review period, the WEP draft is discussed, improved, and updated based on incoming feedback from the community. Feedback is welcome from the whole community, as long as it arrives within the review period. 
+
+#. `Final review and resolution`_ — At the end of the review period, the WEP is examined for **lazy consensus**. A lazy consensus means a situation where the community either provided feedback, approved the WEP explicitly, or ignored the WEP altogether. In some cases where consensus wasn't reached, the review period might be extended, but never shortened.
+
+#. `Implementation`_ — If the WEP is accepted, the processes or policies described in the WEP are implemented by the CoreOps team, the WEP author, or specific community members as defined in the WEP.
+
+The following sections describe each stage in more detail.
+
+Pre-proposal
+............
+
+The WEP process begins with a new idea for Write the Docs. It is highly recommended that a single WEP contain a single key proposal or new idea. 
+
+To help prepare the WEP draft, we encourage community members to discuss the idea publicly, for example in the Slack group, and gather initial community feedback before writing a full WEP. Public discussions on pre-proposals also help to make sure the idea is applicable to the entire community and not just to the author.
+
+A WEP can be co-authored by more than one community member, but we recommend keeping the number of co-authors small and choosing a representative who will submit the PR to the WEPs repository on behalf of the co-authors.
```

Comment:
> we recommend keeping the number of co-authors small

I'm not sure we should even mention this. But if you think it's important, I'd like an explanation.

If I were a WEP author, I'd want to have as many "co-sponsors" as possible.

Reply:
There is a difference between "co-authors" and "co-sponsors". Co-authors are actively creating the WEP before it's submitted to community discussion, while co-sponsors should be enlisted during the discussion phase. The goal here is to distinguish ownership of the WEP creation and feedback implementation, and you probably don't want 10 people authoring the same WEP for logistical reasons :)

Reply:
I think we stole this from the PEP process, but I think the goal is to have a couple people responsible for shepherding it. If 5 people come together it becomes hard to figure out who is in charge and can make decisions. I think smaller is definitely better in terms of the proposal team, but not in terms of who looks at and reviews it.

Reply:
Appreciate the explanation. Then maybe s/on behalf of the co-authors/on behalf of all supporters of the WEP/

(The current text, to me, implies that support is limited to the authors)

Reply:
I wrote in a suggestion to expand this paragraph and differentiate between authors and supporters. Feel free to edit/adapt as needed.